TU-Agent/buildConstruction.test2g

```diff
@@ -3,18 +3,24 @@ use agentKnowledge as knowledge.
 use agentGoals as goals.
 use buildConstruction as module.
 
-test tygronagent with
+test buildConstruction with
 	%check if there is at least an oldbuilding believe.
-	pre {true}
+	pre { bel(availableLandPolygon(MP)) }
 	in {
 		%these actions should never be called within the demolish module
 		never done(building_plan_demolish(_)).
 		never done(map_buy_land(_, _)).
 		never done(map_sell_land(_, _, _)).
 		never done(building_plan_upgrade(_, _)).
+		
+		always done(building_plan_construction(_, _, MP)), not(bel(availableLandPolygon(MP))).
```

Comment:
If you are in the buildConstruction module, you always build right?
If you don't want to build, you don't want to go to the build module